src/settings/Settings.js

```diff
@@ -372,4 +372,10 @@ export const SETTINGS = {
         default: false,
         controller: new LowBandwidthController(),
     },
+    "fallbackICEServerAllowed": {
+        supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS,
+        displayName: _td("Allow fallback call assist server turn.matrix.org"),
```

Comment:
This might need ab it of wordsmithing - it's moderately unclear what this does in isolation.

Reply:
I have updated it to be more detailed.